toolboxes/nfft/gpu/cuGriddingConvolution.cu

```diff
@@ -12,6 +12,8 @@
 #include "ConvolverNC2C_sparse.cuh"
 #include "ConvolverNC2C_standard.cuh"
 
+#include <thrust/host_vector.h>
```

Comment:
Is this needed?

Reply:
Nope, that's left over from the workaround before the thrust fix. Thanks for catching it.